exercises/concept/luigis-luscious-lasagna/.docs/hints.md

```diff
@@ -0,0 +1,19 @@
+# General
+
+- The let expressions is described in the [Elm lang documentation][let-elm-lang].
+- Whitespace is important
+- There can only be one expression in the `in` section
+
+This example shows the syntax
+
+```elm
+cylinderArea : Float -> Float -> Float
+cylinderArea radius height =
+    let
+        sideArea = 2 * pi * radius * height
+        topArea = pi * radius ^2
+    in
+        sideArea + 2 * topArea
+```
```

Comment:
I'm not sure this will render well on the website. We usually try to not have code in the hints, but to link to external resources instead (which you do in the top of the file).

Reply:
Ok, I've deleted this.